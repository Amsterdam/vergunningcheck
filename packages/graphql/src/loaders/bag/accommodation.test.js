require("@spotify/polly-jest-presets");
const { loaderFunctionsTest } = require("../../testUtils");

const accommodation = require("./accommodation");

const key1 = "0363010000915417";
const key2 = "0363010000782985";

describe("Accommodation loader", () => {
  describe("Basic functions", loaderFunctionsTest(accommodation, key1, key2));
});
