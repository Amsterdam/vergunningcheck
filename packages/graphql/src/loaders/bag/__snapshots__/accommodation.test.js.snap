// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Accommodation loader Basic functions module result 1`] = `
Array [
  Object {
    "lat": 121263,
    "lon": 484745,
    "mainAddressNationalId": "0363200000372662",
  },
]
`;

exports[`Accommodation loader Basic functions module result 2`] = `
Array [
  Object {
    "lat": 121607,
    "lon": 487996,
    "mainAddressNationalId": "0363200000409571",
  },
]
`;
